apps/src/netsim/NetSimLobby.html.ejs

```diff
@@ -7,6 +7,7 @@
   <div id="shard_view">
     <label for="netsim_shard_select">My Section:</label>
     <select id="netsim_shard_select"></select>
+    <span id="netsim_not_connected_note">You are not connected. Select a class section from the list to connect.</span>
```

Comment:
Yes, this is pretty lame; there's already some UI pain around the lobby component, I think I'd like to wait until I break it apart to do something different with this "disconnected" message.